mdagent/tools/base_tools/simulation_tools/setup_and_run.py

```diff
@@ -683,7 +687,23 @@ def setup_system(self):
         self.pdb_path = self.path_registry.get_mapped_path(self.pdb_id)
         self.pdb = PDBFile(self.pdb_path)
         self.forcefield = ForceField(*self.params["forcefield_files"])
-        self.system = self._create_system(self.pdb, self.forcefield, **self.sys_params)
+        try:
+            self.system = self._create_system(
+                self.pdb, self.forcefield, **self.sys_params
+            )
+            print("System built successfully")
+            print(self.system)
+        except ValueError as e:
+            if "No template found for" in str(e):
+                raise ValueError(str(e))
+            else:
+                print(
+                    f"Error building system. Please check the forcefield \
+                      files {str(e)}"
+                )
+                raise ValueError(
+                    f"Error building system. Please check the forcefield files {str(e)}"
+                )
```

Comment:
I'm not sure if I understand the purpose of this block. Basically, there's two ValueErrors, but why state "Error building system" twice, in line 700 then line 705? Is it because our MDAgent doesn't always print out the Observation step?

Reply:
oh, it can be confusing youre right. This will happen in case the residue is not part of the solvent list, then it will raise an error. The print statement before the second raise was basically debugging. Ill get rid of it :)